invenio_rdm_records/services/config.py

```diff
@@ -225,3 +225,50 @@ class RDMFileDraftServiceConfig(FileServiceConfig):
         "content": FileLink("{+api}/records/{id}/draft/files/{key}/content"),
         "commit": FileLink("{+api}/records/{id}/draft/files/{key}/commit"),
     }
+
+
+class SecretLinkServiceConfig(RecordServiceConfig):
```

Comment:
moderate: The config is 100% the same than the `RecordServiceConfig`? If so there is no need to create it... You can instantiate the subservice by passing the `RecordServiceConfig`.